deploy/complete/kubernetes/README.md

```diff
@@ -11,8 +11,26 @@ Refer to [helm-chart](../helm-chart/README.md)
 > NOTE: There are several options for ingress controllers in K8S. This demo uses the common `ingress-nginx`
 
 ```text
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/mandatory.yaml
```

Comment:
We should not alter this and hardcode a specific version. We already have an effort to always support the latest.

Reply:
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/mandatory.yaml
doesnt work anymore. so had to change.

took it from here: https://docs.cloud.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengsettingupingresscontroller.htm
we can also use kube-proxy to forward to edge. but optionally can use ingress also. that was my intent.

Reply:
is this generated by helm ?

Reply:
yes. The main deployment are on deploy/complete/helm-charts. deploy/complete/kubernetes are generated from there.

The setup chart that takes care of the ingress, and has specific configurations to support Grafana and other services. We do not use the suggested standard from the documentation.

Reply:
And we nginx 0.30 is deprecated, and has some documented issues. Even for other examples, you should be using 3.x+ instead of 0.x